app/views/transactions/_transaction.html.slim

```diff
@@ -1,4 +1,4 @@
-tr.transaction id=dom_id(transaction) class=colorize_amount(transaction.amount) data-edit-url=edit_transaction_path(transaction)
+tr.transaction id=dom_id(transaction) class=colorize_amount(transaction.amount) data-edit-url=edit_transaction_path(transaction) class=colorize_transaction(transaction)
```

Comment:
@Gataniel please don't use `colorize_amount` here, use `colorize_transaction` helper only and set color for positive and negative transactions in this helper too.